### DISCUSSION TITLE
User can choose which alias to receive newsletter

### DISCUSSION THREAD

**Discussion by nguyenkims** [simple-login/app#425] [Feature Requests]
Moved from https://trello.com/c/LAkBNCu4/122-user-can-choose-which-alias-to-receive-newsletter

**Comment by purple-emily** [comment]
@nguyenkims how about as a quick and dirty solution: add a button somewhere on settings that is "subscribe to newsletter" which who'll auto generate a new alias to receive the communications from simple login. 

This should be quick to implement if we just run whatever function runs on user creation to generate the initial alias in the first place?

I know it doesn't fully solve the solution of directly selecting which alias receives the newsletter but it allows people who have deleted the initial alias to recreate it.

**Reply by nguyenkims** [reply]
This can be a good idea indeed. In the meantime user can explicitly opt in for the newsletter on https://app.simplelogin.io/dashboard/setting, the newsletter will be sent to their SimpleLogin account address.

**Reply by purple-emily** [reply]
@nguyenkims I have had that switched on for a long time and don't think I have ever received a newsletter. How often are they?

**Reply by nguyenkims** [reply]
We haven't sent a newsletter for a while.
